### PR TITLE
BRS:325 - Implements FAQ for DUP admin

### PR DESCRIPTION
**Pull request by Christopher-walsh22**
### Jira Ticket:
BRS-325

### Jira Ticket URL:
[Ticket 325](https://github.com/orgs/bcgov/projects/49/views/1?pane=issue&itemId=47055544)

### Description:
Adds FAQ to sidebar if a user is sysadmin
Allows users to access FAQ page if user is a sysadmin
On FAQ page Load retrieves FAQ data to be displayed in WYSIWYG
WYSIWYG edits contents of FAQ data stored in db
Toast notification on success or failure of editing FAQ
Buttons to match Edit park WYSIWYG

### CODE REVIEW

**Review by meredithom — CHANGES_REQUESTED** [review]
Will review again once the issue we discussed with the rest button is sorted out! 
Also small nit to remove unnecessary console logs.

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -51,6 +51,18 @@ const routes: Routes = [
...
       ),
   },
+  {
+    path: 'faq',
+    canActivate: [AuthGuard],
+    component: FaqComponent,
+    data: {
+      label: 'FAQ',
+      breadcrumb: 'Frequently Asked Questions',
+      icon: 'bi-bar-chart',
```
[the remark this answers is not in the document]
Comment (Christopher-walsh22) [reply src/app/app-routing.module.ts]: Addressed in: [Commit](https://github.com/bcgov/parks-reso-admin/pull/332/commits/87a1429390e10ee909de0b551118a4d29694f51f)

**Comment by Christopher-walsh22** [comment]
@meredithom After our talk about the reset button I reached out to @ndelraye and received confirmation the reset button was not needed. I have removed that functionality as well as the console logging you have mentioned. Should be good for re-review now :)

**Review by meredithom — APPROVED** [review]

**Review by marklise — APPROVED** [review]
👍 merge when API side is all ready

**Comment by LindsayMacfarlane** [comment]
Development ticket associated with this PR still open, so leaving PR open.

**Comment by manuji** [comment]
Closing the PR as this work has been already completed and deployed to PROD.

**Pull request merged**